src/components/shared/CardList/Card/Card.jsx

```diff
@@ -26,7 +26,9 @@ export class Card extends React.Component {
     }
 
     render() {
-        const src = this.props.card.cardId ? require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`) : require('../../../../images/cardBack.jpg');
+        let src;        
+        if (this.props.gameMode === 'telltales') src = this.props.card.cardId ? require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`) : require('../../../../images/cardBack.jpg')
```

Comment:
I think having a game mode called "telltales" is a bit confusing. Maybe something like "original"?